### PR TITLE
Add docs.ubuntu.com/core image

### PR DESCRIPTION
**Pull request by WillMoggridge**
Point docs.ubuntu.com/core to new image
This follows the same config as docs.ubuntu.com/phone

## QA

Let's assume you've got [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) installed.

``` bash
# Run minikube
minikube start
minikube addons enable ingress

# Create the service
cat services/docs.ubuntu.com.yaml | sed 's|prod-comms.docker-registry.canonical.com|canonicalwebteam|' | sed 's|[:][$][{]TAG_TO_DEPLOY[_aa-zA-Z]*[}]||' | sed 's|replicas: 5|replicas: 1|' | kubectl apply --filename -

# Create the ingress domains
cat ingresses/staging/docs.staging.ubuntu.com.yaml | sed '/namespace:/d' | kubectl apply --filename -
cat ingresses/production/docs.ubuntu.com.yaml | sed '/namespace:/d' | kubectl apply --filename -

# Wait a couple of minutes

# Test everything with curl
curl -I -H 'Host: docs.ubuntu.com' `minikube ip`  # Should give you 200 OK
curl -I -H 'Host: docs.ubuntu.com' `minikube ip`/core  # Should redirect you to /core/en/
curl -I -H 'Host: docs.ubuntu.com' `minikube ip`/core/en/stacks/  # Should give you 200 OK
curl -I -H 'Host: docs.ubuntu.com' `minikube ip`/core/en/stacks  # Should redirect you to /core/en/stacks/
curl -I -H 'Host: docs.ubuntu.com' `minikube ip`/core/en/stacks/index  # Should redirect you to /core/en/stacks/
curl -I -H 'Host: docs.staging.ubuntu.com' `minikube ip`  # Should give you 200 OK
curl -I -H 'Host: docs.staging.ubuntu.com' `minikube ip`/core  # Should redirect you to /core/en/
curl -I -H 'Host: docs.staging.ubuntu.com' `minikube ip`/core/en/stacks/  # Should give you 200 OK
curl -I -H 'Host: docs.staging.ubuntu.com' `minikube ip`/core/en/stacks  # Should redirect you to /core/en/stacks/
curl -I -H 'Host: docs.staging.ubuntu.com' `minikube ip`/core/en/stacks/index  # Should redirect you to /core/en/stacks/

### CODE REVIEW

File: services/docs.ubuntu.com.yaml [hunk 1]
```diff
@@ -63,6 +78,39 @@ spec:
...
+spec:
+  replicas: 5
+  template:
+    metadata:
+      labels:
+        app: docs.ubuntu.com-core
+    spec:
+      containers:
+        - name: docs-ubuntu-com-core
+          image: prod-comms.docker-registry.canonical.com/docs.ubuntu.com-core:${TAG_TO_DEPLOY_core}
```
Comment (tbille) [inline services/docs.ubuntu.com.yaml]: `TAG_TO_DEPLOY_core` core lower case?

Comment (WillMoggridge) [reply services/docs.ubuntu.com.yaml]: Was following the way `TAG_TO_DEPLOY_phone` was written

Comment (tbille) [reply services/docs.ubuntu.com.yaml]: do we want to keep it that way?
It seems strange to have this var like this...

Comment (nottrobin) [reply services/docs.ubuntu.com.yaml]: It's a fair point. I don't mind about the casing here. But if we change it we need to update the Jenkins jobs.

Comment (tbille) [reply services/docs.ubuntu.com.yaml]: Let's put every global var upper case!

Comment (WillMoggridge) [reply services/docs.ubuntu.com.yaml]: OK, I added a commit

**Pull request merged**